kasparovsyncd/sync.go

```diff
@@ -796,8 +796,10 @@ func updateRemovedChainHashes(dbTx *gorm.DB, removedHash string) error {
 			}
 		}
 
-		dbTransaction.AcceptingBlockID = nil
```

Comment:
I think it might be a good idea to update `dbTransaction.AcceptingBlockID` nonetheless, so that the in-memory objects are in-sync with the DB.
Same for all instances of similar fix.

It's a bit redundant, but can prevent very annoying bugs in the future.

Reply:
Also, near every such .Update() - write short comment about why we are not using .Save() - for the future generations.